api4/plugin.go

```diff
@@ -25,8 +23,6 @@ const (
 )
 
 func (api *API) InitPlugin() {
-	mlog.Debug("EXPERIMENTAL: Initializing plugin api")
-
```

Comment:
@hanzei this is already merged but I'm just curious what your intension was here. Should we have kept the Initializing plugin api" portion?

Reply:
I don't think so. Neither of the other API handlers has a similar log message.